src/components/data-grid/export/docDefinition.ts

```diff
@@ -0,0 +1,250 @@
+import { ColumnApi, GridApi, ValueFormatterParams } from '@ag-grid-community/core';
+import { Content, Margins, TDocumentDefinitions } from 'pdfmake/interfaces';
+import { PdfHeaderCell, PdfTableCell, PrintParams, Translations } from '../types';
+
+export const getDocDefinition = (
+  gridApi: GridApi,
+  columnApi: ColumnApi,
+  printParams: PrintParams,
+  translations: Translations,
+) => {
+  const {
+    PDF_HEADER_COLOR,
+    PDF_INNER_BORDER_COLOR,
+    PDF_OUTER_BORDER_COLOR,
+    PDF_ODD_BKG_COLOR,
+    PDF_EVEN_BKG_COLOR,
+    PDF_HEADER_HEIGHT = 25,
+    PDF_ROW_HEIGHT = 15,
+    PDF_PAGE_ORIENTATION,
+    PDF_WITH_HEADER_IMAGE,
+    PDF_WITH_FOOTER_PAGE_COUNT,
+    PDF_LOGO = '',
+  } = printParams;
+
+  return (function () {
```

Comment:
Is there any reason to return classic func instead of arrow?

Reply:
Nope haha. This is just from the example code that I used. I'll update it